Plugins/Flow.Launcher.Plugin.Program/Programs/UWP.cs

```diff
@@ -377,6 +413,20 @@ private async void Launch(IPublicAPI api)
                 });
             }
 
+            private async void LaunchElevated()
+            {
+                string command = "shell:AppsFolder\\" + UniqueIdentifier;
+                command = Environment.ExpandEnvironmentVariables(command.Trim());
+
+                var info = new ProcessStartInfo(command)
+                {
+                    UseShellExecute = true,
+                    Verb = "runas",
+                };
+
+                Main.StartProcess(Process.Start, info);
```

Comment:
if this works, do we still need to activation helper for uwp app?

Reply:
not really, however, Microsoft's launcher uses the activation helper for normal user launch and this method for elevated launch.